src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    internal static class MergeCommand
+    {
+        public static int Run(MergeOptions mergeOptions)
+        {
+            // Get all SARIF files listed in options.
+            IEnumerable<string> sarifFiles = from path in mergeOptions.Files
```

Comment:
This may be easier to read if it wasn't linq.  Also, each variable being set with these linq statements could be in their own methods.  This would also eliminate the comments above each variable.